spec/FSi/Bundle/DoctrineExtensionsBundle/Resolver/FSiFilePathResolverSpec.php

```diff
@@ -111,7 +117,7 @@ function it_generate_url_with_passed_prefix_even_if_there_is_a_prefix_in_globals
      */
     function it_generate_fsi_file_basename($file)
     {
-        $file->getName()->shouldBeCalled()->willReturn('TestFolder/File/file.jpg');
+        $file->getName()->willReturn('TestFolder/File/file.jpg');
 
         $this->fileBasename($file)->shouldReturn('file.jpg');
```

Comment:
maybe we should test if the filename is encoded here like [here](https://github.com/fsi-open/doctrine-extensions-bundle/pull/45/files#diff-ae6ddbd14596047c574825c79155c0bdR50)